person.rb

```diff
@@ -1,14 +1,15 @@
 require_relative 'corrector'
 
 class Person
-  attr_accessor :name, :age
+  attr_accessor :name, :age, :rentals
   attr_reader :id
 
-  def initialize(age:, name: 'Unknown', parent_permission: true)
+  def initialize(age, name: 'Unknown', parent_permission: true)
     @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] kindly add `has-many` relation between rental and person. By adding a method `add_rental` in Person class like you did before in Book class.
>![image](https://user-images.githubusercontent.com/53402600/185990595-6a62669f-e785-4eec-b279-1749621dd996.png)